modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -149,11 +186,23 @@ def address_combined
     end
 
     def domestic_phone?
+      # TODO: revisit - in what ways are we differentiating between our handling of international vs domestic
       phone_country_code == '1'
     end
 
     def address
-      form_data['address'].presence || {}
+      form_data['address'] || ''
+    end
+
+    def mpi_veteran
```

Comment:
This should guard against being used with a `Claimant` type, and return nil if the appellant is a claimant.